packages/nodes-base/nodes/Note/Note.node.json

```diff
@@ -0,0 +1,24 @@
+{
+	"node": "n8n-nodes-base.note",
+	"nodeVersion": "1.0",
+	"codexVersion": "1.0",
+	"categories": [
+		"Core Nodes"
+	],
+	"resources": {
+		"primaryDocumentation": [
+			{
+				"url": "https://docs.n8n.io/nodes/n8n-nodes-base.note/"
+			}
+		]
```

Comment:
can you remove this? devrel team can add this later when page is ready

Reply:
Removed